resources/assets/js/app/components/MainNavigation.js

```diff
@@ -29,8 +31,10 @@ class MainNavigation extends Component {
     render() {
         const navClass = !this.state.isNavOpen ? 'collapse' : '';
         const dropdownClass = this.state.isDropdownOpen ? 'show' : '';
-        const logInUrl = !!sessionStorage.jwt ? '/logout' : '/login';
-        const logInMsg = !!sessionStorage.jwt ? 'Logout' : 'Login';
+        const logInUrl = isAuthTokenExists() ? '/logout' : '/login';
```

Comment:
create 2 separate component for navigation: 1 for logged in user, 1 for not logged in user. Logged in navigation should contains all available nav items, Logged out nav only login and registration links